playbooks/openstack/README.md

```diff
@@ -202,52 +174,56 @@ $ ansible-playbook --user openshift \
-cluster now.
+* [Configure DNS][configure-dns]
+* [Get the `oc` Client][get-the-oc-client]
+* [Log in Using the Command Line][log-in-using-the-command-line]
+* [Access the UI][access-the-ui]
 
-Take a look at [how to access the cluster][accessing-openshift]
-and [how to remove it][uninstall-openshift] as well as the more
-advanced configuration:
+Read the [Post-Install page][post-install] for a full list of options.
 
-* [Accessing the OpenShift cluster][accessing-openshift]
-* [Removing the OpenShift cluster][uninstall-openshift]
-* Set Up Authentication (TODO)
-* [Multiple Masters with a load balancer][loadbalancer]
-* [External Dns][external-dns]
-* Multiple Clusters (TODO)
-* [Cinder Registry][cinder-registry]
 
+## Uninstall
+
+Everything in the cluster is contained within a Heat stack. To
+completely remove the cluster and all the related OpenStack resources,
+run this command:
+
+```bash
+openstack stack delete --wait --yes openshift.example.com
+```
 
 [ansible]: https://www.ansible.com/
 [openshift-ansible]: https://github.com/openshift/openshift-ansible
 [openshift-ansible-setup]: https://github.com/openshift/openshift-ansible#setup
 [devstack]: https://docs.openstack.org/devstack/
 [tripleo]: http://tripleo.org/
-[ansible-dependencies]: ./advanced-configuration.md#dependencies-for-localhost-ansible-controladmin-node
-[control-host-image]: https://hub.docker.com/r/redhatcop/control-host-openstack/
+[packstack]: https://www.rdoproject.org/install/packstack/
+[configure-authentication]: https://docs.openshift.org/latest/install_config/configuring_authentication.html
 [hardware-requirements]: https://docs.openshift.org/latest/install_config/install/prerequisites.html#hardware
 [origin]: https://www.openshift.org/
 [centos7]: https://www.centos.org/
 [sample-openshift-inventory]: https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example
-[advanced-configuration]: ./advanced-configuration.md
-[accessing-openshift]: ./advanced-configuration.md#accessing-the-openshift-cluster
-[uninstall-openshift]: ./advanced-configuration.md#removing-the-openshift-cluster
-[loadbalancer]: ./advanced-configuration.md#multi-master-configuration
-[external-dns]: ./advanced-configuration.md#dns-configuration-variables
-[cinder-registry]: ./advanced-configuration.md#creating-and-using-a-cinder-volume-for-the-openshift-registry
+[configuration]: ./configuration.md
+[loadbalancer]: ./configuration.md#multi-master-configuration
+[external-dns]: ./configuration.md#dns-configuration
+[cinder-registry]: ./configuration.md#cinder-backed-registry-configuration
+[post-install]: ./post-install.md
+[configure-dns]: ./post-install.md#configure-dns
+[get-the-oc-client]: ./post-install.md#get-the-oc-client
+[log-in-using-the-command-line]: ./post-install.md#log-in-using-the-command-line
+[access-the-ui]: ./post-install.md#access-the-ui
+[scale-deployment]: ./post-install.md#scale-deployment
```

Comment:
This refers to unsupported contrib repo. Let's please remove such references.

Reply:
Ah, missed this one - removed, thanks!